src/packages/ganache/package.json

```diff
@@ -110,9 +114,16 @@
   },
   "bundledDependencies": [
     "@trufflesuite/bigint-buffer",
+    "@trufflesuite/uws-js-unofficial",
+    "@ethereumjs/vm",
+    "@ethereumjs/trie",
+    "@ethereumjs/evm",
     "abstract-leveldown",
+    "async-eventemitter",
     "emittery",
     "keccak",
+    "leveldown",
+    "levelup",
```

Comment:
Do you remember why `levelup` was added? It isn't in develop and the version, 5.1.1, is the same as before.

Reply:
I don't and was just wondering the same thing. I tried removing it and adding it back to `src/chains/ethereum/ethereum` as it is in develop, but I'm getting compilation errors on a project using Ganache:
```
node_modules/ganache/dist/ganache.d.ts:13:25 - error TS2307: Cannot find module 'levelup' or its corresponding type declarations.

13 import { LevelUp } from 'levelup';
```

Reply:
It seems that reverting the api-extractor version has fixed this issue